action.yml

```diff
@@ -41,7 +46,12 @@ runs:
     - name: Run actionlint and format output as JSON
       id: actionlint
       shell: bash
-      run: actionlint -format '{{json .}}' | tee actionlint_output.json
+      run: |
+        # Reference: https://github.com/actions/download-artifact#outputs
+        actionlint \
+          -ignore 'property "download-path" is not defined in object type {}' \
+          -format '{{json .}}' \
+        | tee actionlint_output.json
```

Comment:
You can use `run: >` and get rid of all the backslashes, cf https://yaml-multiline.info/

Note that the comment line has to go outside the `run` block then.

Reply:
This is a backport so I'm keeping it as is for now